cmake/std/atdm/cee-rhel6/custom_builds.sh

```diff
@@ -31,12 +31,10 @@ elif atdm_match_any_buildname_keyword \
   # matching of defaults
 
 elif atdm_match_any_buildname_keyword \
-    intel-18.0.2-mpich2-3.2 \
-    intel-18.0.2_mpich2-3.2 \
-    intel-18.0.2 \
-    intel-18 \
```

Comment:
No
```
intel-19.0.3 \
intel-19 \
```

?

Reply:
Removing automerge in case this comment requires a change.

Reply:
@jmgate, the default 'intel-19.0.3' and 'intel-19' maps to `INTEL-19.0.3_INTELMPI-2018.4` and **not** `ATDM_CONFIG_COMPILER=INTEL-19.0.3_MPICH2-3.2`.  See:

* https://github.com/trilinos/Trilinos/blob/afa21bdbacb31dfd7299d1c5db8be33fb9f98dc5/cmake/std/atdm/cee-rhel6/custom_builds.sh#L39
* https://github.com/trilinos/Trilinos/blob/afa21bdbacb31dfd7299d1c5db8be33fb9f98dc5/cmake/std/atdm/cee-rhel6/custom_builds_unit_tests.sh#L139
* https://github.com/trilinos/Trilinos/blob/afa21bdbacb31dfd7299d1c5db8be33fb9f98dc5/cmake/std/atdm/cee-rhel6/custom_builds_unit_tests.sh#L143

Reply:
@jmgate, I think the units in:

* https://github.com/trilinos/Trilinos/blob/afa21bdbacb31dfd7299d1c5db8be33fb9f98dc5/cmake/std/atdm/cee-rhel6/custom_builds_unit_tests.sh

show that this logic matches expected/desired behavior.